### PR TITLE
LPS-130553 | master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-130553

cc @david-gutierrez-mesa 
cc @Tim-Cao

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/misc/iframe/IFrame.testcase [hunk 1]
```diff
@@ -3,22 +3,31 @@ definition {
 
 	property portal.release = "true";
 	property portal.upstream = "true";
-	property testray.component.names = "IFrame Portlet";
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/misc/iframe/IFrame.testcase:6]: Keep it

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/misc/highestratedassets/HighestRatedAssets.testcase [hunk 2]
```diff
@@ -18,92 +23,105 @@ definition {
...
 	@priority = "4"
-	test ViewHighestRatedAsset {
-		JSONGroup.addGroup(groupName = "Site Name");
-
-		for (var n : list "1,2") {
-			JSONLayout.addPublicLayout(
-				groupName = "Site Name",
-				layoutName = "Test Page ${n}");
+	test ViewRatingScore {
+		property testray.component.names = "Asset Publisher";
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/misc/highestratedassets/HighestRatedAssets.testcase:34]: Remove it

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/misc/highestratedassets/HighestRatedAssets.testcase [hunk 3]
```diff
@@ -3,13 +3,18 @@ definition {
 
 	property portal.release = "true";
 	property portal.upstream = "true";
-	property testray.component.names = "Asset Publisher,Highest Rated Assets";
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/misc/highestratedassets/HighestRatedAssets.testcase:6]: Keep it

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/misc/lookandfeel/LookAndFeel.testcase [hunk 4]
```diff
@@ -3,13 +3,18 @@ definition {
 
 	property portal.release = "true";
 	property portal.upstream = "true";
-	property testray.component.names = "Look And Feel";
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/misc/lookandfeel/LookAndFeel.testcase:6]: Keep it

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/misc/lookandfeel/LookAndFeel.testcase [hunk 5]
```diff
@@ -18,72 +23,105 @@ definition {
 		if ("${testPortalInstance}" == "true") {
 			PortalInstances.tearDownCP();
 		}
+		else {
+			JSONGroup.deleteGroupByName(groupName = "Test Site Name");
+		}
 	}
 
+	@description = "View Look and Feel configurations are applied to widgets."
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/misc/lookandfeel/LookAndFeel.testcase:31]: The user could define the look and feel of widgets

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/misc/lookandfeel/LookAndFeel.testcase [hunk 6]
```diff
@@ -18,72 +23,105 @@ definition {
...
 	@priority = "5"
-	test ConfigureCustomStylesViaLookAndFeelPG {
+	test ViewCustomStyles {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/misc/lookandfeel/LookAndFeel.testcase:33]: ConfigureLookAndFeelOfWidget

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/misc/mostviewedassets/MostViewedAssets.testcase [hunk 7]
```diff
@@ -3,13 +3,18 @@ definition {
 
 	property portal.release = "true";
 	property portal.upstream = "true";
-	property testray.component.names = "Asset Publisher,Most Viewed Assets";
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/misc/mostviewedassets/MostViewedAssets.testcase:6]: Keep it

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/misc/mostviewedassets/MostViewedAssets.testcase [hunk 8]
```diff
@@ -18,101 +23,125 @@ definition {
...
 	@priority = "4"
-	test AssertMostViewedAsset {
-		JSONGroup.addGroup(groupName = "Site Name");
-
-		for (var n : list "1,2") {
-			JSONLayout.addPublicLayout(
-				groupName = "Site Name",
-				layoutName = "Test Page ${n}");
+	test ViewCountOfViews {
+		property testray.component.names = "Asset Publisher";
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/misc/mostviewedassets/MostViewedAssets.testcase:34]: Remove it

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/misc/mostviewedassets/MostViewedAssets.testcase [hunk 9]
```diff
@@ -18,101 +23,125 @@ definition {
 		if ("${testPortalInstance}" == "true") {
 			PortalInstances.tearDownCP();
 		}
+		else {
+			JSONGroup.deleteGroupByName(groupName = "Test Site Name");
+		}
 	}
 
+	@description = "The user can view the number of views on the web content via Most Viewed Assets widget."
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/misc/mostviewedassets/MostViewedAssets.testcase:31]: The assets should be ordered by view counts in Most Viewed Assets widget.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/misc/mostviewedassets/MostViewedAssets.testcase [hunk 10]
```diff
@@ -18,101 +23,125 @@ definition {
...
 	@priority = "4"
-	test AssertMostViewedAsset {
-		JSONGroup.addGroup(groupName = "Site Name");
-
-		for (var n : list "1,2") {
-			JSONLayout.addPublicLayout(
-				groupName = "Site Name",
-				layoutName = "Test Page ${n}");
+	test ViewCountOfViews {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/misc/mostviewedassets/MostViewedAssets.testcase:33]: ViewOrderOfAssets

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/misc/nestedportlets/NestedPortlets.testcase [hunk 11]
```diff
@@ -3,13 +3,18 @@ definition {
 
 	property portal.release = "true";
 	property portal.upstream = "true";
-	property testray.component.names = "Nested Portlets";
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/misc/nestedportlets/NestedPortlets.testcase:6]: Keep it

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/misc/nestedportlets/NestedPortlets.testcase [hunk 12]
```diff
@@ -18,50 +23,61 @@ definition {
...
 		}
+		else {
+			JSONGroup.deleteGroupByName(groupName = "Test Site Name");
+		}
 	}
 
+	@description = "User can nest a widget inside Nested Applications widget."
 	@priority = "5"
-	test ViewWCDNestedPortlets {
+	test ViewNestedWebContentDisplay {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/misc/nestedportlets/NestedPortlets.testcase:33]: NestWidgetInsideNestedApplications

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/misc/sitemap/SiteMap.testcase [hunk 13]
```diff
@@ -33,22 +28,36 @@ definition {
...
 	}
 
+	@description = "View private parend and child page via Site Map widget."
 	@priority = "4"
-	test DisplayPrivatePageWithChildPage {
-		JSONLayout.addWidgetToPrivateLayout(
-			groupName = "Test Site Name",
-			layoutName = "Child Page Name",
-			widgetName = "Site Map");
+	test ViewPrivateParentAndChildPage {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/misc/sitemap/SiteMap.testcase:33]: DisplayPrivateParentAndChildPage

File: portal-web/test/functional/com/liferay/portalweb/macros/SiteMapPortlet.macro [hunk 14]
```diff
@@ -1,43 +1,12 @@
 definition {
 
-	macro viewCustomADTMultiColumnPG {
-		for (var pageName : list "${listPageName}") {
-			var key_pageName = "${pageName}";
-
+	macro viewPagePG {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/SiteMapPortlet.macro:3]: viewPageItem

**Review by Tim-Cao — COMMENTED** [review]
Hi @ozysouza, Could you update according to comments? Thanks! Would you mind send another PR for last 5 commits? I will review it deeply.

**Comment by ozysouza** [comment]
> Hi @ozysouza, Could you update according to comments? Thanks! Would you mind sending another PR for last 5 commits? I will review it deeply.

Hi @Tim-Cao, I've updated the PR according your comments. I'll send the PR with the others changes once this here be merged, If a send now it'll have conflicts.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/misc/highestratedassets/HighestRatedAssets.testcase [hunk 15]
```diff
@@ -18,92 +24,103 @@ definition {
...
+	@description = "View rating score in Highest Rated Assets widget."
 	@priority = "4"
-	test ViewHighestRatedAsset {
-		JSONGroup.addGroup(groupName = "Site Name");
-
-		for (var n : list "1,2") {
-			JSONLayout.addPublicLayout(
-				groupName = "Site Name",
-				layoutName = "Test Page ${n}");
+	test ViewRatingScore {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/misc/highestratedassets/HighestRatedAssets.testcase:34]: Keep the original name

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/misc/mostviewedassets/MostViewedAssets.testcase [hunk 16]
```diff
@@ -18,101 +24,123 @@ definition {
...
+	@description = "The assets should be ordered by view counts in Most Viewed Assets widget."
 	@priority = "4"
-	test AssertMostViewedAsset {
-		JSONGroup.addGroup(groupName = "Site Name");
-
-		for (var n : list "1,2") {
-			JSONLayout.addPublicLayout(
-				groupName = "Site Name",
-				layoutName = "Test Page ${n}");
+	test ViewOrderOfAssets {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/misc/mostviewedassets/MostViewedAssets.testcase:34]: ViewMostViewedAsset

**Comment by Tim-Cao** [comment]
@ozysouza, See changes https://github.com/liferay-echo/liferay-portal/pull/7074/commits/8e7be71c52fbbd623aff8ded825aa83ee33d9b2e

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**